test/NNPDE_tests.jl

```diff
@@ -484,3 +485,68 @@ u_predict  = [first(phi(x,res.minimizer)) for x in xs]
 
 # plot(xs ,u_real, label = "analytic")
 # plot!(xs ,u_predict, label = "predict")
+
+## Example 8, Lorenz System (Parameter Estimation)
+println("Example 8, Lorenz System")
+
+Random.seed!(1234)
+@parameters t ,σ_ ,β, ρ
+@variables x(..), y(..), z(..)
+Dt = Differential(t)
+eqs = [Dt(x(t)) ~ σ_*(y(t) - x(t)),
+       Dt(y(t)) ~ x(t)*(ρ - z(t)) - y(t),
+       Dt(z(t)) ~ x(t)*y(t) - β*z(t)]
+
+
+bcs = [x(0) ~ 1.0, y(0) ~ 0.0, z(0) ~ 0.0]
+domains = [t ∈ IntervalDomain(0.0,1.0)]
+dt = 0.05
+
+function lorenz!(du,u,p,t)
+ du[1] = 10.0*(u[2]-u[1])
+ du[2] = u[1]*(28.0-u[3]) - u[2]
+ du[3] = u[1]*u[2] - (8/3)*u[3]
+end
+
+u0 = [1.0;0.0;0.0]
+tspan = (0.0,1.0)
+prob = ODEProblem(lorenz!,u0,tspan)
+sol = solve(prob, Tsit5(), dt=0.1)
+function getData(sol)
+    data = []
+    indx = rand(1:1:21 , 12)
+    for i in indx
+        data = vcat(data , (sol.u[i] , sol.t[i]))
+    end
+    return data
+end
+data = getData(sol)
+#Additional Loss Function
+function additional_loss(phi, θ , p)
```

Comment:
θ = [flat_θ; estim_param] 
so it's needed here to not grab all param and only flat_θ

```julia
initθs = DiffEqFlux.initial_params.([chain1,chain2,chain3])
acum =  [0;accumulate(+, length.(initθs))]
sep = [acum[i]+1 : acum[i+1] for i in 1:length(acum)-1]
len = length(data)
function additional_loss(phi, θ , p)
    _loss(u , t) = sum(abs2, phi[i](t , θ[sep[i]])[1] - u[i]  for i in 1:1:3)
    global data
    return sum(abs2, _loss(u,t) for (u , t) in data)/len
end
```